src/modules/users/components/TokenActivation/TokenActivationContent/TokensTab.tsx

```diff
@@ -0,0 +1,213 @@
+  activeTokens: BigNumber;
+  inactiveTokens: BigNumber;
+  totalTokens: BigNumber;
+  lockedTokens: BigNumber;
+  isPendingBalanceZero: boolean;
+  token: UserToken;
+  colonyAddress: Address;
+}
+
+const TokensTab = ({
+  activeTokens,
+  inactiveTokens,
+  totalTokens,
+  lockedTokens,
+  token,
+  colonyAddress,
+  isPendingBalanceZero,
+}: TokensTabProps) => {
+  const targetRef = useRef<HTMLParagraphElement>(null);
+
+  const [totalTokensWidth, setTotalTokensWidth] = useState(0);
+
+  const widthLimit = 164;
+
+  useLayoutEffect(() => {
+    if (targetRef?.current && totalTokensWidth === 0) {
+      setTotalTokensWidth(targetRef?.current?.offsetWidth);
+    }
+  }, [totalTokensWidth]);
+
+  const hasLockedTokens = useMemo(() => !lockedTokens.isZero(), [lockedTokens]);
+
+  const tokenDecimals = useMemo(
+    () => getTokenDecimalsWithFallback(token?.decimals),
+    [token],
+  );
+
+  const formattedTotalAmount = formatTokenValue({
+    value: totalTokens,
+    suffix: ` ${token?.symbol}`,
+    unit: tokenDecimals,
+    truncate: 3,
+  }).split(' ')[0];
+
+  return (
+    <>
+      <div className={styles.totalTokensContainer}>
+        <div
+          className={
+            totalTokensWidth <= widthLimit
+              ? styles.tokenSymbol
+              : styles.tokenSymbolSmall
+          }
+        >
+          <TokenIcon
+            token={token || {}}
+            size={totalTokensWidth <= widthLimit ? 'xs' : 'xxs'}
+          />
+        </div>
+        <p
+          ref={targetRef}
+          className={
+            totalTokensWidth <= widthLimit
+              ? styles.totalTokens
+              : styles.totalTokensSmall
+          }
+        >
+          {formattedTotalAmount} <span>{token?.symbol}</span>
+        </p>
+      </div>
+      <div className={styles.tokensDetailsContainer}>
+        <ul>
+          <li>
+            <TokenTooltip
+              className={styles.listItemActive}
+              content={<FormattedMessage {...MSG.activeTokensTooltip} />}
+            >
+              <FormattedMessage
+                {...(!hasLockedTokens ? MSG.active : MSG.activeLocked)}
+              />
+            </TokenTooltip>
+            <div className={styles.tokenNumbers}>
+              <Numeral
+                value={activeTokens}
+                suffix={` ${token?.symbol}`}
+                unit={tokenDecimals}
+                truncate={3}
+              />
+            </div>
+            <TokenTooltip
+              className={styles.lockedTokens}
+              content={<FormattedMessage {...MSG.stakedTokensTooltip} />}
+            >
+              <FormattedMessage {...MSG.staked} />
+            </TokenTooltip>
+            <div className={styles.tokenNumbersLocked}>
+              <Numeral
+                value={lockedTokens}
+                suffix={` ${token?.symbol}`}
+                unit={tokenDecimals}
+                truncate={3}
+              />
+            </div>
```

Comment:
This should be a separate list entry, why is it bundled with the "active" list entry ?

Reply:
It was under the active in my mind, but you are right. I've corrected it.